docs/source/changelog.rst

```diff
@@ -55,15 +45,21 @@ New Features
 Enhancements
 ++++++++++++
 
+- (:pr:`62`, :pr:`67`, :pr:`83`) A large block of TeraChem improvements thanks to @ffangliu contributions.
+  Changed the input parser to call qcelemental to_string method with bohr unit, improved output of parser to turn stdout
+  into Result, and modified how version is parsed.
 - (:pr:`63`) QCEngine functions ``util.which``, ``util.which_version``, ``util.parse_version``, and
   ``util.safe_version`` removed after migrating to QCElemental.
+- (:pr:`65`) Torchani can now handle the ANI1-x and ANI1-ccx models. Credit to @dgasmith for implementation
+- (:pr:`74`) Removes caching and reduces pytorch overhead from Travis CI. Credit to @dgasmith for implementation
 - (:pr:`77`) Rename ``ProgramExecutor`` to ``ProgramHarness`` and ``BaseProcedure`` to ``ProcedureHarness``.
 - (:pr:`77`) Function ``util.execute(..., outfiles=[])`` learned to collect output files matching a globbed filename.
 - (:pr:`81`) Function ``util.execute`` learned list argument ``as_binary`` to handle input or output
   files as binary rather than string.
 - (:pr:`81`) Function ``util.execute`` learned bool argument ``scratch_exist_ok`` to run in a preexisting directory.
   This is handy for stringing together execute calls.
 - (:pr:`84`) Function ``util.execute`` learned str argument ``scratch_suffix`` to identify temp dictionaries for debugging.
+- (:pr:`90`) DFTD3 now supports parameters for zero and Becke-Johnson damping to use with SAPT0-D
```

Comment:
if you make further edits, I'd add the modifier "preliminary" to parameters. fitting not final.

`v0.7` or `v0.7.0`? https://github.com/samuelcolvin/pydantic/releases

Reply:
If we are following the Semantic Version guide https://semver.org/ linked in slack, then it would be v0.7.0